compiler/dependencies/runtimeClasspath.txt

```diff
@@ -5,9 +5,9 @@ com.squareup:kotlinpoet:1.15.3
 javax.annotation:jsr250-api:1.0
 javax.inject:javax.inject:1
 org.jetbrains.intellij.deps:trove4j:1.0.20200330
-org.jetbrains.kotlin:kotlin-compiler-embeddable:1.9.20
-org.jetbrains.kotlin:kotlin-daemon-embeddable:1.9.20
+org.jetbrains.kotlin:kotlin-compiler-embeddable:1.9.22
+org.jetbrains.kotlin:kotlin-daemon-embeddable:1.9.22
 org.jetbrains.kotlin:kotlin-reflect:1.9.21
-org.jetbrains.kotlin:kotlin-script-runtime:1.9.20
-org.jetbrains.kotlin:kotlin-stdlib:1.9.21
+org.jetbrains.kotlin:kotlin-script-runtime:1.9.22
+org.jetbrains.kotlin:kotlin-stdlib:1.9.22
```

Comment:
I wonder if it's worth adding the Kotlin bom to keep this `kotlin-reflect` dependency in line with the others.

Reply:
:+1: sounds good to me, opened a separate draft: https://github.com/square/anvil/pull/853

RIP to dokka's kotlin-reflect v1.6.10 in compiler-api 😆